tests/python/pants_test/base/test_exception_sink.py

```diff
@@ -26,6 +26,7 @@ def test_default_log_location(self):
     pants initialization, and when pants runs this test, it won't have been reset to the fake
     buildroot we use for tests.
     """
+    ExceptionSink.reset_log_location(os.path.join(os.getcwd(), '.pants.d'))
```

Comment:
The comment above is no longer accurate.

But also, I would suggest deleting this test, since `test_reset_log_location` does the same thing.

Reply:
Deleted `test_default_log_location`